File: event-listener/src/main/java/com/acme/events/java/EventListenerImpl.java
package com.acme.events.java;

import io.cloudevents.CloudEvent;
import io.cloudevents.jackson.JsonFormat;
import io.infinitic.cloudEvents.CloudEventListener;
import java.util.List;

public class EventListenerImpl implements CloudEventListener {
    @Override
    public void onEvents(List<CloudEvent> cloudEvents) {
        for (CloudEvent event : cloudEvents) {
            System.out.println(new String(new JsonFormat().serialize(event)));
        }
    }
}

File: event-listener/src/main/kotlin/com/acme/events/kotlin/EventListenerImpl.kt
package com.acme.events.kotlin

import io.infinitic.cloudEvents.CloudEventListener
import io.cloudevents.CloudEvent
import io.cloudevents.jackson.JsonFormat

@Suppress("unused")
class EventListenerImpl : CloudEventListener {
    override fun onEvents(cloudEvents: List<CloudEvent>) {
        cloudEvents.forEach { event ->
            println(String(JsonFormat().serialize(event)))
        }
    }
}
